packages/integration-karma/test/template/attribute-capture/index.spec.js

```diff
@@ -0,0 +1,12 @@
+import { createElement } from 'lwc';
```

Comment:
This PR doesn't offer any special behavior with the `capture` attribute. Can we rename the test to give it a more representative name?